.gitignore

```diff
@@ -17,4 +18,4 @@ coverage
 # editors
 .idea/*
 *.iml
-*.sublime-*
\ No newline at end of file
+*.sublime-*
```

Comment:
Missed it myself, sorry. But I don't think I added that line? Must have been EOL.